packages/loopring_v3/contracts/aux/bridge/IBridgeConnector.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2017 Loopring Technology Limited.
+pragma solidity ^0.7.0;
+pragma experimental ABIEncoderV2;
+
+import "./BridgeData.sol";
+
+
+interface IBridgeConnector
+{
+    function processCalls(ConnectorCalls calldata connectorCalls)
```

Comment:
I suggest we do not pass-in `ConnectorCalls`, but `ConnectorGroup[]`. Many fields of `ConnectorCalls` should be transparent to `IBridgeConnector`

Reply:
I suggest we move the definitation of `ConnectorCalls` or whatever parameters taken by `processCalls` here into this contract to make it eaiser for people to understand the parameter struct without visiting other files. If `IBridgeConnector` is easy to understand, it's going to be less error-prone for implementation.

`ConnectorCalls` is currently hard to understand, and people will have to go to the Bridge.sol file to understand it's fields.